llama-index-integrations/vector_stores/llama-index-vector-stores-oracledb/llama_index/vector_stores/oracledb/base.py

```diff
@@ -95,25 +96,23 @@ def _escape_str(value: str) -> str:
 column_config: Dict = {
     "id": {"type": "VARCHAR2(64) PRIMARY KEY", "extract_func": lambda x: x.node_id},
     "doc_id": {"type": "VARCHAR2(64)", "extract_func": lambda x: x.ref_doc_id},
-    "text": {
-        "type": "CLOB",
-        "extract_func": lambda x: _escape_str(
-            x.get_content(metadata_mode=MetadataMode.NONE) or ""
-        ),
+    "embedding": {
+        "type": "VECTOR",
+        "extract_func": lambda x: array.array("f", x.get_embedding()),
```

Comment:
* Unable to insert nodes with a high embedding dim.
  * Currently, we cast the list with the `float` values to a string and then insert the data into the `VECTOR` typed column in the Oracle.
  * Behind the scenes, this string is assigned to the temporary variable with type `VARCHAR2`, before being stored as a `VECTOR`
  * VARCHAR2 has a max size of 32,767. When string representation with a bigger size is stored, [the error is raised](https://docs.oracle.com/en/error-help/db/ora-01461/?r=23ai)
  * Wrapping the array into `array.array`, not converting into a string, [as suggested in oracledb doc](https://python-oracledb.readthedocs.io/en/latest/user_guide/vector_data_type.html#inserting-float32-float64-and-int8-vectors), resolves the issue